content/zones/frozen-lakes.json

```diff
@@ -0,0 +1,58 @@
+loadout: basicFoundation
+startingItems: [
+  copper/300
+  lead/450
+  metaglass/300
+  graphite/450
+]
+configureObjective: {
+  type: Launched
+  zone: frozen-lake
+}
+requirements: [
+  {
+    type: Launched
+    zone: derelict-desert-outpost
+  }
+  {
+    type: unlock
+    block: iron-conveyor.json
```

Comment:
Why do you put .json after iron-conveyor here?

Reply:
Lol sorry, can you fix that for me? I have some urgencies to do

Reply:
Or do you want me to do it a bit later

Reply:
Aight I'll fix it